std/range/package.d

```diff
@@ -6269,15 +6269,22 @@ FrontTransversal!(RangeOfRanges, opt) frontTransversal(
 }
 
 // Bugzilla 16442
-@safe unittest
+pure @safe nothrow unittest
 {
     int[][] arr = [[], []];
 
     auto ft1 = frontTransversal!(TransverseOptions.assumeNotJagged)(arr);
     assert(ft1.empty);
 
-    auto ft2 = frontTransversal!(TransverseOptions.enforceNotJagged)(arr);
-    assert(ft2.empty);
+    try
```

Comment:
I didn't realize this was not a DDOC'd unit test, so it's not as big of a deal, but I still don't like this and don't really see much of a point.

Reply:
The test can be split into a separate `unittest` block if it can't be `nothrow`.